apps/sms/views/inbox/js/startup.js

```diff
@@ -39,6 +40,7 @@
       ConversationClient.init(App.instanceId);
       MessageManager.init();
       Navigation.init();
+      Drafts.init();
```

Comment:
Yeah I think so. Maybe I can left some comment in bug 1080504, I think it's the most appropriate bug for addressing this issue.